### DISCUSSION TITLE
Your GitHub Copilot subscription has ended.

### DISCUSSION THREAD

**Discussion by jordidimass** [community/community#51784] [Copilot Conversations]

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @jordidimass ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by jordidimass** [comment]
Thank you

**Comment by Akash1134** [comment]
Hi @jordidimass , Could you please provide more details on the message you're getting? Which version of Copilot and IDE are you using and what subscription type do you have?
